tests/test_BlkDiagMatrix.py

```diff
@@ -378,6 +378,36 @@ def test_blk_diag_to_diag(self):
         """
         self.assertTrue(np.allclose(np.diag(self.blk_a.dense()), self.blk_a.diag()))
 
+    def test_from_dense(self):
+        """
+        Test truncating dense array returns correct block diagonal entries.
+        """
+        B = BlkDiagMatrix.from_dense(self.dense, self.blk_partition)
+
+        self.allallfunc(B, self.blk_a)
+
+    def test_from_dense_warns(self):
+        """
+        Test truncating dense array returns correct block diagonal entries,
```

Comment:
Ah, thanks. Yeah it did before, I simplified the test to just check the warning, since it was a dupicate of the code above it. I missed updating the docstring.  Fixed the string. Thanks